datalad/support/annexrepo.py

```diff
@@ -3508,15 +3509,15 @@ def _proc_json_record(self, j):
                     # do not crash if no command is reported
                     label=j['action'].get('command', '').capitalize(),
                     unit=' Bytes',
-                    total=float(j['total-size']),
+                    total=float(j.get('total-size', None)),
```

Comment:
I guess that should also be `s/None/0/` as it is done below to prevent `float(None)`, right?

Reply:
yes, I guess should be `0` as well as we are not having special handling for None

```suggestion
                    total=float(j.get('total-size', 0)),
```